nexus/reconfigurator/planning/src/planner.rs

```diff
@@ -4773,50 +4923,8 @@ pub(crate) mod test {
         // attached.
         let target_release_generation = Generation::from_u32(2);
 
-        // Manually specify a trivial TUF repo.
```

Comment:
I removed this part of the test because we now get planner errors if we try to plan an update with a TUF repo that contains no artifacts. Specifically: it will check whether Nexus can be updated, and fail because it can't determine whether any other zones are updated (because the TUF repo has no artifacts for them).